src/components/chip/chip.stories.tsx

```diff
@@ -16,6 +16,15 @@ export default {
       options: ['small', 'large'],
       control: 'radio',
     },
+    icon: {
+      control: 'select',
+      options: ['none', 'check', 'time'],
+      mapping: {
+        none: null,
+        check: <CheckCircleOutline />,
+        time: <AccessTime />,
+      },
+    },
```

Comment:
Do you think this gives the impression that these are the only 2 icons that can be used with the Chip?